tools/browserutils/commands.json

```diff
@@ -0,0 +1,2 @@
+{"install-browser": {"path": "install.py", "script": "run", "parser": "get_parser", "help": "Install browser components",
```

Comment:
`wpt install-browser firefox browser` looks a bit redundant, while `wpt
install-browser firefox webdriver` is a little misleading. What do you think
about calling this sub-command `install` instead?